packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,32 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## January
 
+### Weekly release 2022.01.2
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Device information in the OAuth 2.0 interact flow](#device-information-in-the-oauth-2-0-interact-flow) | January 26, 2022 |
+| [Fewer digits revealed for shorter phone numbers](#fewer-digits-revealed-for-shorter-phone-numbers) | January 26, 2022 |
+| [Bugs fixed in 2022.01.2](#bugs-fixed-in-2022-01-2)         | January 26, 2022           |
+
+#### Device information in the OAuth 2.0 interact flow
```

Comment:
We don't document the /interact API for OIE? Should this be in the release notes?